uint32 no_seri_control_unit

string rfid_tid

uint8 golongan_kendaraan

string message